### DISCUSSION TITLE
Download image size

### DISCUSSION THREAD

**Discussion by gemiren** [coil-kt/coil#874] [Q&A] [answered]
I'd like to download an image of it's original size if it's smaller than my target size or the target size if the image's original size is larger. My target is not an ImageView. How can do that? Thanks.

**Comment by colinrtwhite** [comment, marked as answer]
You can do this by setting `size(width, height)` and `precision(Precision.INEXACT)` when building your image request.

**Reply by gemiren** [reply]
Awesome! That's exactly what I'm looking for. Thanks.
